### PR TITLE
Adding new supported states to Ios_interfaces resource module

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

Adding new supported states to Ios_interfaces resource module

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

ios_interfaces

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
rechek

File: plugins/modules/ios_interfaces.py [hunk 1]
```diff
@@ -71,17 +72,29 @@
         - full
         - half
         - auto
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the VyOS device by executing
```
Comment (NilashishC) [inline plugins/modules/ios_interfaces.py:78]:
```suggestion
      - The value of this option should be the output received from the IOS device by executing
```

**Comment by justjais** [comment]
recheck

File: plugins/modules/ios_interfaces.py (hunk 1 above)
Comment (justjais) [reply plugins/modules/ios_interfaces.py]: updated

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Pull request merged**